services/galley/src/Galley/API/Update.hs

```diff
@@ -366,29 +373,75 @@ updateLocalConversationMessageTimer ::
   Local ConvId ->
   Public.ConversationMessageTimerUpdate ->
   Galley (UpdateResult Event)
-updateLocalConversationMessageTimer lusr zcon lcnv update = do
+updateLocalConversationMessageTimer lusr con lcnv update =
+  getUpdateResult $
+    updateLocalConversation lcnv (unTagged lusr) (Just con) $
+      ConversationActionMessageTimerUpdate update
+
+-- | Update a local conversation, and notify all local and remote members.
+updateLocalConversation ::
+  Local ConvId ->
+  Qualified UserId ->
+  Maybe ConnId ->
+  ConversationAction ->
+  MaybeT Galley Event
+updateLocalConversation lcnv qusr con action = do
+  -- retrieve conversation
   (conv, self) <-
-    getConversationAndMemberWithError
-      (errorDescriptionTypeToWai @ConvNotFound)
-      (lUnqualified lusr)
-      (lUnqualified lcnv)
+    lift $
+      getConversationAndMemberWithError
+        (errorDescriptionTypeToWai @ConvNotFound)
+        qusr
+        (lUnqualified lcnv)
 
   -- perform checks
-  ensureActionAllowedThrowing ModifyConversationMessageTimer self
-  ensureGroupConvThrowing conv
```

Comment:
Is this handled elsewhere?

Reply:
Yes, by [`ensureConversationActionAllowed`](https://github.com/wireapp/wire-server/pull/1801/files#diff-b5ce77f9fddb8d607ffb7e48175549469ace834031d1a95d6b85776f1df71adbR143) in [`updateLocalConversation`](https://github.com/wireapp/wire-server/pull/1801/files#diff-259be1b8909f30421ab02e60765d6e90ae68426b95c3aa8303c57945dc84dd37R382).